source/install/desktop.rst

```diff
@@ -71,3 +69,23 @@ To download the unofficial, community-driven .deb packages for the desktop appli
 2. Open a terminal and execute the command ``sudo dpkg -i mattermost-desktop-<VERSION>-<ARCH>.deb``
 
 3. Open **Dash** (located at top left corner) and input ``mattermost``, then click the Mattermost icon. The Mattermost desktop should open automatically.
+
+Troubleshooting
+--------------------------------------------------
+
+Possible solutions to issues encountered when using the Desktop App.
+
+"Installation has failed" dialog
```

Comment:
Cool, I changed this to a definition list.

We might want to visit other docs that use headers for troubleshooting section for consistency (and maybe add to doc guidelines?)